ui/public/i18n/language.json

```
@@ -120,6 +127,7 @@
         "IgnoreIP": "All IP in this list will never be banned by Fail2ban (One IP per line)",
         "MailJailState": "All relative events to the jail (start or stop), will be notified to root and other recipients",
         "FindTime": "When fail2ban starts, the service looks for attackers in the logs for this time interval",
-        "LogLevel": "The DEBUG level could generate huge log size and fill your disk space"
+        "LogLevel": "The DEBUG level could generate huge log size and fill your disk space",
+        "BanTime_Incremental": "The incremental ban time allows to increase the time of ban by a multiplier (2,4,6,8,10,20,30,60,300,720,1440,2880) after each faillure found in log. It allows you to set a short ban time (60 seconds for example), a valid user can be banned for a short time but a brute force bot will be banned for a long time quickly. The recidive jail is disabled by the Incremental BanTime"
```

```suggestion
        "BanTime_Incremental": "The incremental ban time allows to increase the time of ban after each failure found in log. It allows you to set a shorter ban time, a valid user can be banned for a short time but a brute force attacker will be banned for a long time. The recidive jail is disabled by the Incremental BanTime"
```

You say that `The recidive jail is disabled by the Incremental BanTime`, but I can't find recidive configuration inside the UI